lib/gollum/templates/navbar.mustache

```diff
@@ -71,6 +71,7 @@
           </button>
           <a
             class="btn BtnGroup-item btn-sm hide-sm hide-md"
+            id="minibutton-history"
             href="{{edit_path}}/{{escaped_url_path}}"
             id="minibutton-edit-page"
```

Comment:
I believe it is invalid for HTML elements to have two `id`s defined. So if they are both actively being used (in JavaScript) we will either have to consolidate the functionality to one ID or use classes instead.

Reply:
I searched for `minibutton-history` in the project files and it was only found in navbar.mustache, so it is not user in any JS code. I don't remember why I added that id, that must be my mistake. Will remove in the future commit!